source/core/parameters_lagrangian.cc

```diff
@@ -605,21 +606,33 @@ namespace Parameters
                         "1",
                         Patterns::Integer(),
                         "Output frequency");
-      prm.enter_subsection("center of mass coordinate");
-      prm.declare_entry("x",
-                        "0",
-                        Patterns::Double(),
-                        "X coordinate of center of mass");
-      prm.declare_entry("y",
-                        "0",
-                        Patterns::Double(),
-                        "Y coordinate of center of mass");
-      prm.declare_entry("z",
+      prm.declare_entry("center of mass",
+                        "0,0,0",
+                        Patterns::List(Patterns::Double(), 3, 3),
+                        "Coordinate of center of mass");
+      prm.declare_entry("moment of inertia",
+                        "0,0,0",
+                        Patterns::List(Patterns::Double(), 3, 3),
+                        "Boundary inertia around x,y,z axis");
+      prm.declare_entry("triangulation mass",
                         "0",
                         Patterns::Double(),
-                        "Z coordinate of center of mass");
-      prm.leave_subsection();
+                        "Triangulation mass in kg");
 
+      prm.enter_subsection("triangulation motion");
```

Comment:
The same comment here: maybe it's better to call this type of motion "free motion"